tests/test_ci/TestControllers.py

```diff
@@ -14,7 +15,138 @@
                         generate_signature, mock_api_request_github)
 
 
+class MockKVM:
+
```

Comment:
Wouldn't pylint return a error if the Class doesn't have documentation?

Reply:
Same

Reply:
No, since we are ignoring DocString errors currently in the testing module. It requires a lot of rework everywhere before we can implement `pydocstyle` here. Also since this is only for developers who contribute, and they do not need to understand all tests, a single line docstring with just the test is enough.